web_switch_company_warning/static/src/js/switch_company_warning.js

```diff
@@ -0,0 +1,50 @@
+openerp.web_switch_company_warning = function (instance) {
+
```

Comment:
Can you use ``"use strict";`` as [recommended ](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#javascript)?